src/Tempest/Console/Testing/ConsoleTester.php

```diff
@@ -85,7 +85,7 @@ public function call(string|Closure|array $command): self
 
                 $attribute = $handler->getAttribute(ConsoleCommand::class);
 
-                if (! $attribute) {
+                if ($attribute === null) {
```

Comment:
needed to run Rector otherwise I couldn't push 😅 or let the checks in this PR pass